lib/timeline/component/ItemSet.js

```diff
@@ -1946,13 +1946,34 @@ class ItemSet extends Component {
         fullNestedGroups = fullNestedGroups.concat(nextLevel);
       }
     }
-    let nestedGroups = groupsData.get(fullNestedGroups).map(function (nestedGroup) {
+    var nestedGroups;
+    if(nestingGroup.showNested){
+      var showNestedGroups=groupsData.get(nestingGroup.nestedGroups);
+        let i=0;
+        while (i<showNestedGroups.length){
+          let group=showNestedGroups[i];
+          if(group.nestedGroups && group.nestedGroups.length>0 && (group.showNested == undefined || group.showNested == true)){
+            showNestedGroups.push(...groupsData.get(group.nestedGroups));
+          }
+          i=i+1;
+        }						
```

Comment:
Should be a for loop.

Reply:
Thank you @mojoaxel, same has been done.